$OpenBSD$

Generating the info documentation errors out:

sgmltmp.grc65.2.texi:96:
    Cross reference to nonexistent node `example-grc' (perhaps incorrect sectioning?).
sgmltmp.grc65.2.texi:41:
  Cross reference to nonexistent node `building-seq' (perhaps incorrect sectioning?).

--- doc/Makefile.orig	Wed Mar 25 16:24:45 2015
+++ doc/Makefile	Sat Apr  4 01:21:10 2015
@@ -43,7 +43,7 @@ endif
 zip:
 	@cd .. && zip cc65 html/*.*
 
-doc: html info
+doc: html
 
 html: $(addprefix ../html/,$(SGMLS:.sgml=.html) doc.css doc.png)
 
